### DISCUSSION TITLE
OpenAPI for UERANSIM similar to `srs/corescope` for protocol testing of 5G-Core Network

### DISCUSSION THREAD

**Discussion by prpr2770** [aligungr/UERANSIM#463] [Ideas]
Is it possible to develop an OpenAPI interface for UERANSIM, similar to the way [SRSRAN/CORESCOPE](https://github.com/srsran/corescope) have implemented - so that it might be possible to avail the larger set of features that's provided for NAS and NGAP protocols within UERANSIM?

**Comment by aligungr** [comment]
Hi @prpr2770

Thanks for the suggestion. It seems like a good idea.

And I'd like to know what kind of APIs you'd like. Could you give some examples for more clarification?
